test/check-application

```diff
@@ -253,6 +253,49 @@ class TestApplication(testlib.MachineCase):
 
         self.allow_journal_messages("/run/podman/io.podman: couldn't connect.*")
 
+    def testRunImage(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/podman")
+        b.wait_present("#containers-images")
+        b.wait_in_text("#containers-images", "busybox:latest")
+
+        # Open run image dialog
+        b.wait_present('#containers-images tr:contains("busybox:latest")')
+        b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-actions button')
+        b.wait_visible('div.modal-dialog div.modal-header h4.modal-title:contains("Run Image")')
```

Comment:
Is that dialog already present, but hidden? The react-y way would be for it to be not rendered at all when closed. Then this should be `wait_present`.

Reply:
It should be hidden.